_posts/brice_aminou/2017-03-07-Mobile-app-with-Laravel.md

```diff
@@ -13,4 +13,123 @@ tags:
 teaser:
     info: An easy way to create a mobile app from an existing Laravel application
 ---
-This tutorial describes an easy way to create a mobile app from your existing Laravel application.
+This tutorial describes an easy way to create a mobile app from your existing Laravel application. At the end of this tutorial, you will be able to create a hybrid app from an
```

Comment:
I did not want to talk too much about Laravel, but the post is answering a problem that I had using Laravel and that I was not able to find the explanation online. So I don't want to make any recommendations or comparisons to other tools as there is already a lot of different existing posts comparing the different frameworks.